### PR TITLE
Added DbContext

### PR DESCRIPTION
**Pull request by PukarSubedi01**
Added DbContext

### CODE REVIEW

File: SilverCarRental/SilverCarRental.Core/Interface/IRepository.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SilverCarRental.Core.Interface
+{
+    public interface IRepository<T> where T : class
+    {
+        IEnumerable<T> GetAll();
+        T GetById(int id);
```
Comment (srigorrepati) [inline SilverCarRental/SilverCarRental.Core/Interface/IRepository.cs:12]: Improve the code by writing fetch all

**Review by rohanudas97 — APPROVED** [review]
nice

**Review by iambissnuu — APPROVED** [review]
Ok

**Review by durapd — APPROVED** [review]

**Comment by bypillaraghu** [comment]
Nice

**Review by bypillaraghu — COMMENTED** [review]
OK

**Pull request merged**